src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -230,7 +230,7 @@ private bool FeaturesEnabled()
                     return true;
                 }
 
-                return false;
+                return true;
```

Comment:
This turns OOP on all the time, regardless of any options.  SHouldn't this check the Tools\Option for this instead?